spec/genernal-entity/general.spec.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable max-classes-per-file */
+import { HttpStatus, INestApplication } from '@nestjs/common';
+import { Controller, Injectable, Module } from '@nestjs/common';
+import { ApiHideProperty, ApiProperty } from '@nestjs/swagger';
+import { Test, TestingModule } from '@nestjs/testing';
+import { InjectRepository, TypeOrmModule } from '@nestjs/typeorm';
+import { Type } from 'class-transformer';
+import { IsNumber, IsOptional, IsString } from 'class-validator';
+import request from 'supertest';
+import { Entity, Repository, Column, DeleteDateColumn, PrimaryGeneratedColumn, CreateDateColumn, UpdateDateColumn } from 'typeorm';
+
+import { Crud } from '../../src/lib/crud.decorator';
+import { CrudService } from '../../src/lib/crud.service';
+import { CrudController, GROUP } from '../../src/lib/interface';
+import { TestHelper } from '../test.helper';
+
+@Entity('general')
+class GeneralEntity {
```

Comment:
It seems like `GeneralEntity` is duplicated in here and `spec/genernal-entity/general.entity.ts`.